src/Security/SecurityControllerBuilder.php

```diff
@@ -80,4 +87,45 @@ public function addLogoutMethod(ClassSourceManipulator $manipulator): void
         );
         $manipulator->addMethodBuilder($logoutMethodBuilder);
     }
+
+    public function addFormLoginMethod(ClassSourceManipulator $manipulator, string $loginTemplatePath): void
+    {
+        $methodBuilder = $manipulator->createMethodBuilder('login', Response::class, false);
+        $methodBuilder->addAttribute($manipulator->buildAttributeNode(Route::class, ['path' => '/login', 'name' => 'app_login']));
+
+        $this->addUseStatements($manipulator, [Route::class]);
+
+        $methodBuilder->addParam((new Param('authenticationUtils'))->setType('AuthenticationUtils'));
```

Comment:
Will there already be a `use` statement for this? In `addLoginMethod()`, we explicitly add the use statement.